From ea55137cc4a565a2917267dcdd62cc7d53493135 Mon Sep 17 00:00:00 2001
From: Kevin Wolf <kwolf@redhat.com>
Date: Tue, 11 Feb 2014 16:24:42 +0100
Subject: [PATCH 25/28] qemu-progress: Drop unused include

RH-Author: Kevin Wolf <kwolf@redhat.com>
Message-id: <1392135884-10508-2-git-send-email-kwolf@redhat.com>
Patchwork-id: 57221
O-Subject: [RHEL-7.0 qemu-kvm PATCH 1/3] qemu-progress: Drop unused include
Bugzilla: 997878
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
Reviewed-by: Benoit Canet <benoit@irqsave.net>
(cherry picked from commit e69968d472bd020a08c677c814237548090d2e59)

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
---
 util/qemu-progress.c | 1 -
 1 file changed, 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 util/qemu-progress.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/util/qemu-progress.c b/util/qemu-progress.c
index 9a3f96c..ad33fee 100644
--- a/util/qemu-progress.c
+++ b/util/qemu-progress.c
@@ -24,7 +24,6 @@
 
 #include "qemu-common.h"
 #include "qemu/osdep.h"
-#include "sysemu/sysemu.h"
 #include <stdio.h>
 
 struct progress_state {
-- 
1.7.1

